ruby/vampires.rb

```diff
@@ -0,0 +1,135 @@
+######################################################################
+
+#vampire intake survey
+
+#######################################################################
+
+num_employees_processed = true
+employee_name = true
+employee_age = true
+year_born = true
+order_garlic = true
+wants_health_insurance = true
+
+
+num_employees_processed = 0
+
+print "How many employees are you processing today?"
+num_employees_processed = gets.chomp.to_i
+#num_employees_processed += 1
+ 
+while 0 != num_employees_processed.to_i
+    puts "lets continue with the Survey."
+    num_employees_processed += -1
+    
+    puts "What is your name?"
+    employee_name = gets.chomp
+    
+    puts "How old are you?"
+    employee_age = gets.chomp.to_i
+
+    puts "What year were you born?"
+    year_born = gets.chomp.to_i
+
+  if

```

Comment:
Remember to let your files read like a story.  You should be checking your conditions on the same line.

```
if condition
  # code
elsif condition
  # code
else
  #code
end
```